lib/minheap.js

```diff
@@ -11,18 +11,30 @@ class MinHeap {
   }
 
   // This method adds a HeapNode instance to the heap
-  // Time Complexity: ?
-  // Space Complexity: ?
+  // Time Complexity: O(log n) (because of helper function I think?)
+  // Space Complexity: O(log n)
   add(key, value = key) {
```

Comment:
👍 , Your space complexity is O(1) however because you didn't use recursion in `heapUp`